app/models/space.rb

```diff
@@ -0,0 +1,16 @@
+class Space
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String
+  property :description, Text
+  property :rate, Integer
+  property :max_capacity, Integer
+  property :available_from_date, Date
+  property :available_to_date, Date
+
+  def is_available?(start_date, end_date)
```

Comment:
Edit this out , write tests for it, put it in the waffle board under suggestsions?